Dockerfile

```
@@ -82,22 +82,60 @@ RUN chmod +x /opt/runpod/scripts/*.sh
+        0|false|no|off|'')
+            printf "false"
+            return 0
+            ;;
+        *)
+            printf "false"
+            return 1
+            ;;
+    esac
+}
+EOF
+RUN chmod +x /usr/local/bin/normalize_flag.sh
```

Hadolint (2.14.0)

[error] unexpected 'R'
expecting a new line followed by the next instruction

(DL1000)